package/Control.roc

```diff
@@ -0,0 +1,111 @@
+module [Control, toCode]
+
+import Style exposing [Style]
+
+# Control (commonly known as Control Sequence Introducer or CSI) represents the control sequences for terminal commands.
+# The provided commands are common and well-supported, though not exhaustive.
+
+Control : [
```

Comment:
Can we make this a Doc comment please `##`. Also can you include a link to a good reference (maybe wikipedia?).

Also can you check the generated docs look ok.

Reply:
done